src/main/java/gov/cms/madie/terminology/service/FhirTerminologyService.java

```diff
@@ -140,10 +140,11 @@ public List<ValueSetForSearch> searchValueSets(String apiKey, Map<String, String
                     ValueSetForSearch.builder()
                         .title(vs.getTitle())
                         .author(
-                            String.valueOf(
-                                vs.getExtensionByUrl(
-                                        "http://hl7.org/fhir/StructureDefinition/valueset-author")
-                                    .getValue()))
+                            Optional.ofNullable(
+                                    vs.getExtensionByUrl(
+                                        "http://hl7.org/fhir/StructureDefinition/valueset   -author"))
```

Comment:
I'm seeing some funky whitespace between `valueset` and `-author`.

Reply:
I think my cat might have co authored.